### PR TITLE
fix typo in tutorial

### PR DESCRIPTION
**Pull request by jimmychu0807**

### CODE REVIEW

File: docs/tutorials/build-permission-network/launch-network.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Now we have 3 well known nodes.
 
 ## Add an extra connection
 
-Let's add another node, but not well known, only a "sub-node" of Charlie, which can only conneted to Charlie's well known node.
+Let's add another node, not a well-known node but a "sub-node" of Charlie, which can only connect to Charlie's well-known node.
```
Comment (riusricardo) [inline docs/tutorials/build-permission-network/launch-network.md:132]: we should stop using the term well-know here. Same as the other tutorial where we mention them.
Well-known keys are a different thing than development keys/users.

Comment (kaichaosun) [reply docs/tutorials/build-permission-network/launch-network.md:132]: Here the well-known node means that the default allowed connections in a permissioned network. 
https://github.com/paritytech/substrate/blob/master/frame/node-authorization/src/lib.rs#L108

**Review by kaichaosun — APPROVED** [review]
LGTM

**Review by danforbes — APPROVED** [review]

File: docs/tutorials/build-permission-network/launch-network.md (hunk 1 above)
Comment (riusricardo) [reply docs/tutorials/build-permission-network/launch-network.md:132]: Yeah I agree with you @kaichaosun, but that is misleading. We shouldn't be using the same term for different definitions.
In summary it is incorrect to use well-known here.

**Pull request merged**